spec/relish_encryption_helper_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper'
+require 'fernet/legacy'
+
+describe Relish::EncryptionHelper do
+  before do
+  end
+
+  describe '#decrypt' do
+    it 'raises RelishDecryptionFailed on incorrect decryption secret' do
```

Comment:
these two tests seem very similar except for raising the `MultiJson::ParseError` maybe the setup portion could be extracted into the `before` block or into `let` statements?